docs/chymyst02.md

```diff
@@ -386,7 +386,7 @@ run { case result(n) + done(x) =>
     }
 ```
 
-To make the chemistry clearer, we can rewrite this as three reactions using pattern-matching:
+To make the chemistry clearer, we can rewrite this as three reactions using pattern matching:
```

Comment:
I don't like too much that compile-time can be compile time and run-time runtime or run time, but... Anyhow, you call the shots on those. I am just laying out my proposal.